pkg/storage/format/formatprovider.go

```diff
@@ -0,0 +1,131 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package format
+
+import (
+	"fmt"
+
+	"github.com/hyperledger/aries-framework-go/pkg/storage"
+)
+
+const (
+	failOpenUnderlyingStore      = "failed to open underlying store: %w"
+	failCloseUnderlyingStore     = "failed to close underlying store: %w"
+	failCloseAllUnderlyingStores = "failed to close all underlying stores: %w"
+	failFormatValue              = "failed to format value: %w"
+	failPutInUnderlyingStore     = "failed to put encrypted document in underlying store: %w"
+	failParseFormattedValue      = "failed to parse formatted value: %w"
+	failDeleteInUnderlyingStore  = "failed to delete key-value pair in underlying store: %w"
+)
+
+// Formatter represents a type that can encrypt and decrypt between
+// Structured Documents and Encrypted Documents.
+type Formatter interface {
```

Comment:
Move this one level up into `storage` package

Reply:
Done.